packages/schema-extract/README.md

```diff
@@ -0,0 +1,246 @@
+# Schema Extract
+
+
+Schema extract transforms code files into [JSON-Schema](https://json-schema.org/) schemas. Schema extraction is a necessary step to create automated documentation for your code.
+
+It is composed of two parts, the **ts transformer** and **linker**.
+
+
+## Playground
+The [schema-playground](../schema-playground) is a playground that allows live editing of your code and transforms it to a schema and displays the results as you type it.
+
+## Usage
+By default, the linker is connected to our TS transformer, it uses it to transforms the code and then link it together.  
```

Comment:
"which uses it to transform"*

Reply:
I would also specify that it's transforming the types of the code (from my understanding) and not necessarily reflecting the logic.